invenio_vocabularies/contrib/funders/datastreams.py

```diff
@@ -38,36 +39,62 @@ def apply(self, stream_entry, **kwargs):
         """Applies the transformation to the stream entry."""
         record = stream_entry.entry
         funder = {}
+        funder["title"] = {}
 
         funder["id"] = normalize_ror(record.get("id"))
         if not funder["id"]:
             raise TransformerError(_("Id not found in ROR entry."))
 
-        funder["name"] = record.get("name")
+        aliases = []
+        acronym = None
+        for name in record.get("names"):
+            lang = name.get("lang", "en")
+            if "ror_display" in name["types"]:
+                funder["name"] = name["value"]
+            if "label" in name["types"]:
+                funder["title"][lang] = name["value"]
+            if "alias" in name["types"]:
+                aliases.append(name["value"])
+            if "acronym" in name["types"]:
+                # We only save the first acronyn, to maintain
+                # compatability with existing data structure
+                if not acronym:
+                    acronym = name["value"]
```

Comment:
minor: could we maybe add the extra acronyms to `aliases`?

Reply:
Ah, that's a good idea. Added!